src/backend/distributed/connection/connection_configuration.c

```diff
@@ -444,7 +444,17 @@ GetEffectiveConnKey(ConnectionHashKey *key)
 		return key;
 	}
 
+	if (!CitusHasBeenLoaded())
```

Comment:
The comment of this function is now even more outdated than it already was. Let's remove the "task tracker" mention from the function comment and replace it with something about the Citus non-main database.